### ISSUE TITLE
Feature request: make bridging over infill selectablebased on infill density.

### ISSUE THREAD

**Issue by wolfmanjm** [alexrj/Slic3r#1986]
Currently the first solid layer on top of an infill layer is always done as a bridge at bridging speed and bridging flow rate.

This is unnecessary if the infill density is around 30% and really slows the print down if the surface is large.

A nice feature would be to use normal solid layer speed and flow rate as the first solid layer if the infill was over a certain selectable percentage.

**Comment by lordofhyphens** [comment]
Yeah, I can see where this would be useful, and it looks straightforward to implement. I'll have a branch + pull request up shortly if someone wants to give a hand testing :)

**Comment by alexrj** [comment]
Hm, I had actually addressed this issue in 38a9e32a287c4a601e256ebd96533f68c0c8dcc0 (after this issue report). And I have one more improvement for making this logic even stricter.
